feat: Added `apiary_extra_tags_s3` variable to add s3 buckets extra tags. (#282)
ExpediaGroup/apiary-data-lake: VARIABLES.md

> | apiary\_tags                                              | Common tags that get put on all resources.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | `map(any)`               | n/a                                                                                                                                                                                                                                                                                            |   yes    |
> | apiary\_extra\_tags\_s3                                   | Extra tags that get put on apiary_s3_logs_bucket.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | `map(any)`               | n/a                                                                                                                                                                                                                                                                                            |  no      |

Reviewer: ```suggestion
| apiary\_tags                                              | Common tags that are added to all resources.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | `map(any)`               | n/a                                                                                                                                                                                                                                                                                            |   yes    |
| apiary\_extra\_tags\_s3                                   | Extra tags that are added to apiary_s3_logs_bucket.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | `map(any)`               | n/a                                                                                                                                                                                                                                                                                            |  no      |
```
